tests/bdd/volgroup_vgpattern_bdd.feature

```diff
@@ -0,0 +1,156 @@
+
+Feature: Validate vgpattern and volgroup based scheduling.
+  Background:
+      Given Single lvmnode cluster is configured
+      And lvm vg "lvmvg" is created on a node
+
+      Scenario: test sc with volgroup present on the node
+          Given a sc is created with volgroup as "lvmvg"
+          And pvc is created referencing the same sc
+          Then lvmvolume cr should get created
+          And cr state should be Ready
+
+      Scenario: test sc with volgroup not present on the node
+          Given a sc is created with volgroup as "lvmvgn"
+          And pvc is created referencing the same sc
+          Then pvc should be in Pending
+          And lvmvolume cr should not get created
+
+  
+  Background:
+      Given Single lvmnode cluster is configured
+      And lvm vg "lvmvg" and "lvmvg1" is created on a node
+
+      Scenario: test sc with vgpattern matching atleast one vg
+          Given sc is created with vgpattern "lvmvg1.*"
```

Comment:
The dot `.` will not match the vg on the node I think. It has to be `lvmvg1*`

Reply:
I followed documentation. It does match. tried it manually.